Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:16
+
+WORKDIR /app
+
+COPY package*.json ./
+
+RUN npm ci
+
+COPY . .
```

Comment:
You're copying everything here. Do a multi-stage build (1 for the TS and the other for running JUST compiled JS)